.travis.yml

```diff
@@ -57,7 +57,7 @@ addons:
 
 branches:
  only:
-   - master
```

Comment:
Why these changes are here? Perhaps, you would need to rebase.

Reply:
I left a note regarding this at the top of PR - 

> PS: I have also modified .travis.yml to build my branch. I plan to remove this when the PR is near merge.

travis builds are very helpful and help checking if any engine code breaks, i can just drop this commit when near-merge